assets/app/views/modals/debug-terminal.html

```diff
@@ -34,7 +34,15 @@ <h2 ng-if="debugPod.status.phase !== 'Failed'" class="text-muted">
       </div>
       <div ng-if="container | entrypoint : image" class="original-cmd-msg">
         <label>Original Command:</label>
-        <code>{{container | entrypoint : image}}</code>
+        <code>
+          <truncate-long-text
+              content="container | entrypoint : image"
+              limit="80"
+              newline-limit="1"
+              expandable="false"
```

Comment:
hrm yeah i guess that could be problematic unless you give it a fixed
width/height and make it scroll, guess this is ok for now, i think majority
of them will be one-liners, all of our ootb ones are i believe @bparees

On Thu, Apr 14, 2016 at 2:49 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/modals/debug-terminal.html
> <https://github.com/jwforres/origin/pull/51#discussion_r59770888>:
>
> > @@ -34,7 +34,15 @@ <h2 ng-if="debugPod.status.phase !== 'Failed'" class="text-muted">
> >        </div>
> >        <div ng-if="container | entrypoint : image" class="original-cmd-msg">
> >          <label>Original Command:</label>
> > -        <code>{{container | entrypoint : image}}</code>
> > +        <code>
> > +          <truncate-long-text
> > +              content="container | entrypoint : image"
> > +              limit="80"
> > +              newline-limit="1"
> > +              expandable="false"
>
> Is there enough room in a dialog for a really long command? You can hover
> your mouse and see it for what it's worth.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jwforres/origin/pull/51/files/55ea8e65cf4ae1390645c204bafaaf3094b65bc0#r59770888>
>